guessing_game_class3/README.md

```diff
@@ -0,0 +1,10 @@
+# Guessing game class-03
+
+# Added a fourth variable to guess my son's age.
+# 4th variable tells user if guess was too high or low.
+# Stretch uses WHILE followed by if/else. 
+
+# Added a counter to keep track of correct answers
+# Counter is given a value of 0. Then, inside each IF statement, counter++ is used to keep track of only correct answers.
+
+# Future fix for final counter alert: my alert says "questions" even if it should be singular. Should I use an if/else statement here?
```

Comment:
Try using bullet points and not using all headers. Let me know if you'd like to see an example of a README. Usually it contains the title and author of the project and then the content and functionality of the project (user stories).